.travis.yml

```diff
@@ -1,18 +1,22 @@
-sudo: required 
-dist: trusty 
+sudo: required
+dist: trusty
 language: generic
 notifications:
   email:
     on_success: change
     on_failure: always
 env:
   global:
-    - ROS_DISTRO="indigo"
     - UPSTREAM_WORKSPACE=file
-    - ROSINSTALL_FILENAME=.travis.rosinstall
   matrix:
-    - ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
+    - ROS_DISTRO="indigo" ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
+    - ROS_DISTRO="jade" ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
+    - ROS_DISTRO="kinetic" ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
```

Comment:
@ipa-mdl don't we need to define `ROSINSTALL_FILENAME` for each job? Or is industrial_ci expecting a `.travis.rosinstall.distro` filename?